guides/v2.3/extension-dev-guide/routing.md

```diff
@@ -165,6 +165,20 @@ $this->_forward('action', 'controller', 'Other_Module')
 {: .bs-callout .bs-callout-tip }
 Use the [`ActionFactory`] in your router to create an instance of an `Action` class.
 
+{: .bs-callout .bs-callout-info }
+Action class should return a `result object`.
+
+## Result object
```

Comment:
Please apply above edits to this table as well except where noted below.

Reply:
**done**